docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
     image: postgres:9.6-alpine
     networks:
       - internal_network
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres"]
```

Comment:
why should we specify these explicitly? shouldn't these be part of the postgres docker image?